pyproject.toml

```diff
@@ -61,9 +61,6 @@ dsp-tools = "dsp_tools.dsp_tools:main"
 # plugin (https://github.com/keattang/poetry-exec-plugin) to define commands available for the developers
 # e.g. `poetry exec check-links`
 check-links = "markdown-link-validator ./docs -i \\.\\/assets\\/.+ -i .+github\\.com\\/dasch\\-swiss\\/dsp-tools\\/settings"
-# `mkdocs serve` can normally be executed without custom port. This is only necessary on machines where the standard
-# port is taken already
-docs-serve = "mkdocs serve --dev-addr=localhost:7979"
```

Comment:
This was only necessary on my old machine. Now I can execute `mkdocs serve` like everyone else, so this script is not necessary any more.